### PR TITLE
Get actual test for reproducer role

### PR DESCRIPTION
**Pull request by cjeanner**
This patch activate a test for one of the reproducer features:
layout reproducer.

Since it's the base feature of the reproducer role, it's a good idea to
ensure it's working properly.

As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running

### CODE REVIEW

**Comment by cjeanner** [comment]
Need some actual tests/validations, of course.

File: ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml [hunk 1]
```diff
@@ -119,14 +120,19 @@
...
+    crc_private_key: >-
+      {{
+        (_layout.vms.crc.image_local_dir, "id_ecdsa") |
+        path_join
+      }}
   block:
     - name: Copy authorized_keys
-      ansible.builtin.command:
+      ansible.builtin.shell:
         cmd: >-
```
Comment (pablintino) [inline ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml:131]: Hard-coding the user here is making my eye twitch. But, I understand that is OK for now

Comment (cjeanner) [reply ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml:131]: In this case I don't think we hit any issue. crc (and OCP actually) have that admin user.

**Pull request merged**